test/util.ts

```diff
@@ -0,0 +1,38 @@
+import {assert} from 'chai';
```

Comment:
An example of the new, declarative unit test format provided by `mocha-typescript`.

For the curious, it *is* possible to do fancier things like parameterized tests. It would look something like this:

```typescript
import {assert} from 'chai';
import {suite, test} from "mocha-typescript";
import * as util from "../src/util";

[
    {input: undefined, output: false},
    {input: null, output: false},
    {input: 0, output: true},
    {input: 1, output: true},
].forEach(({input, output}) => {
    @suite class GeneratedUtilTest {
        @test(`_hasValue(${input})`) test() {
            assert.equal(util._hasValue(input), output);
        }
    }
});

```

IMHO it's not worth it for small parameter sets like the above because:
1. This involves creating a new `@suite` (`GeneratedUtilTest`) for every test, polluting the testing report.
2. The IDE isn't able to run individual tests via a button press when you use runtime dynamism like this.

It's also possible to directly make calls against the `mocha` object, but it has similar gotchas.